kafka/service.go

```diff
@@ -135,15 +24,28 @@ type Consumer struct {
 	readers map[string]*kafka.Reader // topic -> reader mapping
 }
 
-type Security struct {
+type ClientCredentials struct {
 	Username string
 	Password string
 }
 
+type Environment int
+
+const (
```

Comment:
here the environment is int, maybe better to use stringer package to introduce readability for logging.